### PR TITLE
Update code cli command

### PR DESCRIPTION
**Pull request by jbeard4**
Add CLI command to update-code, and consolidated duplicate code

### CODE REVIEW

File: werkit/aws_lambda/deploy.py [hunk 1]
```diff
@@ -76,44 +64,57 @@ def pif(x):
...
-    path_to_zipfile, function_name, s3_code_bucket=None, verbose=True,
+def perform_update_code(
+    path_to_zipfile, function_name, s3_code_bucket=None, verbose=False,
 ):
+    common_args = {"FunctionName": function_name}
+
+    def handler(code_arguments):
+        boto3.client("lambda").update_function_code(**code_arguments, **common_args)
+
+    create_or_update_lambda(
```
Comment (paulmelnikow) [inline werkit/aws_lambda/deploy.py:88]: I'd suggest providing the argument names (i.e. `path_to_zipfile=path_to_zipfile`). It's a little verbose, but it makes this more forgiving of changes to the parameter list, and makes args like the string arg easier to read.

File: werkit/aws_lambda/deploy.py [hunk 2]
```diff
@@ -76,44 +64,57 @@ def pif(x):
...
-            pif("Lambda function created")
-    else:
-        with open(path_to_zipfile, "rb") as f:
-            zipfile_contents = f.read()
-        pif(f"Uploading {path_to_zipfile} to Lambda")
-        boto3.client("lambda").create_function(
-            Code={"ZipFile": zipfile_contents}, **common_args
-        )
-        pif("Lambda function created")
+    def handler(code_arguments):
```
Comment (paulmelnikow) [inline werkit/aws_lambda/deploy.py:67]: Calling this `handler` is confusing because it makes me think it's a Lambda handler function. Maybe `create`?

Comment (jbeard4) [reply werkit/aws_lambda/deploy.py]: Fixed in c2b5794c5b97030d7bf64e4e6c7984842556bbf4

File: werkit/aws_lambda/deploy.py (hunk 1 above)
Comment (jbeard4) [reply werkit/aws_lambda/deploy.py:88]: Fixed in c2b5794c5b97030d7bf64e4e6c7984842556bbf4

**Pull request merged**